configs/rotated_retinanet/rotated-retinanet-rbox-oc_r50_fpn_1x_dior.py

```diff
@@ -0,0 +1,12 @@
+_base_ = 'rotated-retinanet-rbox-le90_r50_fpn_1x_dior.py'
+
+angle_version = 'oc'
+
+model = dict(
```

Comment:
Do we need to update the readme and upload pre-trained models?

Reply:
> Do we need to update the readme and upload pre-trained models?
Yes, I will update the readme and upload these models soon.